tests/subsys/logging/log_list/README.txt

```diff
@@ -0,0 +1,44 @@
+Title: Log list
```

Comment:
You should use .rst for documentation

Reply:
I removed the file. Imo, there is not much value added by keeping this file. Added it because logger-hook had it but i see that most of the tests do not have this file.